src/components/PostList/PostList.jsx

```diff
@@ -1 +1,24 @@
-export const PostList = () => <>Put the list here</>;
+import './PostList.scss';
+
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts, users, comments }) => {
+  const postList = posts.map(post => {
+    const user = users.find(person => person.id === post.userId);
+    const postComments = comments.filter(comment => comment.postId === post.id);
+
+    return {
+      ...post,
+      user,
+      comments: postComments,
+    };
+  });
```

Comment:
Better to do this preparations in `App.jsx`

Reply:
done